templates/common/app_card.html

```diff
@@ -78,7 +78,7 @@ <h5 class="card-title bg-white">{{ app.name }}</h5>
                                 {% else %}
                                     <a title="Launch the app" rel="Link to launch the app" href="{{ app.url }}" target="_blank" class="btn btn-primary">Open</a>
                                 {% endif %}
-                                {% if app.app.slug in 'shinyapp,shinyproxyapp,dashapp,customapp' %}
+                                {% if app.app.slug in 'shinyapp,shinyproxyapp,dashapp,customapp,gradio,streamlit' %}
```

Comment:
Oh, I guess we should add this to this page
https://scilifelab.atlassian.net/wiki/x/AQABrQ

Reply:
Great suggestion, thanks! I now added info about this and more.